source/reference/mongosync.txt

```diff
@@ -161,6 +161,20 @@ source database is dropped at a later time. The ``<db>.system.profile``
 collection will not change the accuracy of user data on the
 destination.
 
+Sharded Clusters
+~~~~~~~~~~~~~~~~
+
+{+c2c-product-name+} supports replication between sharded clusters.
+Individual shards are replicated in parallel from the source cluster to
+the destination cluster, however a :ref:`chunk migration
+<sharding-chunk-migration>` or similar source update could move
+documents to a new source shard during replication.
+
+To preserve event order on the destination cluster,
```

Comment:
We don't "preserve" event order. In fact, we are eventually consistent meaning that the write history/order may not be the same and we only guarantee a consistent snapshot after a healthy cutover. So I would avoid saying "preserve". Instead, I think we just have a mechanism to handle it correctly while maintaining the eventually consistent guarantee.